packages/edit-site/src/components/navigation-sidebar/navigation-toggle/style.scss

```diff
@@ -14,27 +14,18 @@
 
 	body.is-navigation-sidebar-open & {
 		display: flex;
-
-		.edit-site-navigation-toggle__button {
```

Comment:
This was added recently in https://github.com/WordPress/gutenberg/pull/29093/files to address the following issue https://github.com/WordPress/gutenberg/issues/29094. Do you all think we should still keep it?

Reply:
I think something else changed since then that makes this override unnecessary. Here's what I see (using this branch)... looks like border is visible underneath the icon (at least to my eyes)

| Firefox | Safari | Chrome |
| - | - | - |
| <img width="250" alt="Firefox" src="https://user-images.githubusercontent.com/1699996/110869692-8ef11d80-8290-11eb-9694-c8a21684bbeb.png"> | <img width="250" alt="Safari" src="https://user-images.githubusercontent.com/1699996/110869704-97495880-8290-11eb-9d3f-a3e50bafd9c7.png"> | <img width="250" alt="Chrome" src="https://user-images.githubusercontent.com/1699996/110869716-9dd7d000-8290-11eb-8a10-c7a15092b5c2.png"> |